data.js

```diff
@@ -0,0 +1,48 @@
+let Title;
+let Author;
+let bookList = [];
+
+function getValues() {
+  Title = document.querySelector('#title').value;
+  Author = document.querySelector('#author').value;
+  document.querySelector('#title').value = '';
+  document.querySelector('#author').value = '';
+}
+
+function addBook() {
+  document.getElementById('dynamic').innerHTML = '';
+  bookList.forEach((book, index) => {
+    document.getElementById('dynamic').innerHTML += `
+    <div class="books">
+      <p class="title">${book.title}</p>
+      <p class="authir">${book.author}</p>
+      <button class="remove" onClick = 'removeItem(${index})'>Remove</button>
+      <hr> 
+    </div>`;
+  });
+}
+if (localStorage.getItem('Books')) {
+  bookList = JSON.parse(localStorage.getItem('Books'));
+  addBook();
+}
+
+function storeBook() {
+  getValues();
+  const data = {
+    title: Title,
+    author: Author,
+  };
+  bookList.push(data);
+  Title = '';
+  Author = '';
+  localStorage.setItem('Books', JSON.stringify(bookList));
+  addBook();
+}
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please remove the eslint disable rule. Its best to fix the issue in the code.

Reply:
the issue is not fixable without disabling the function. since we have used it in the Addbook function, we cant use it again. its recommended that we disable it
![image](https://user-images.githubusercontent.com/85197908/144031358-b9ee6840-dd6a-4b60-97de-c19b51e902f3.png)